src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -45,23 +45,16 @@ type Props = {
     currentUser?: User,
     getPreviewer: Function,
     hasSkills: boolean,
-    hasProperties: boolean,
+    detailsSidebarProps: Object,
```

Comment:
So the type for this will be DetailsSidebarProps. This type will be exported from DetailsSidebar.js same way I am exporting ContentSidebarBarProps at the bottom of ContentSidebar.js

Reply:
@priyajeet this is causing a few warnings
```warning.js:33 Warning: Failed prop type: The prop `hasClassification` is marked as required in `DetailsSidebar`, but its value is `undefined`.```

```warning.js:33 Warning: Failed prop type: The prop `detailsSidebarProps.hasClassification` is marked as required in `ContentSidebar`, but its value is `undefined`.
    in ContentSidebar```

My change makes sense to me, all of those types have a default and those defaults will get passed down. Not sure why that causes a warning.

Reply:
Actually, these warnings should be here. If the parent doesn't pass in these props warnings should be thrown as they are required (even though they have a default). Since we're giving them a default, why not make them optional?